### PR TITLE
Update to Singularity 2.4 and fix error with Jupyter notebook running as root (sudo)

### PR DESCRIPTION
**Pull request by hypotheses**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ If you haven't installed singularity, do that with [these instructions](http://s
 
 Let's now create a jupyter notebook!
 
-      sudo singularity create --size 4000 jupyter.img
-      sudo singularity bootstrap jupyter.img Singularity
+      sudo singularity image.create --size 4000 jupyter.img
+      sudo singularity build jupyter.img Singularity
```
Comment (vsoch) [inline README.md]: Do we want to remote the image.create command entirely? Is it needed?

Comment (hypotheses) [reply README.md]: When I run the current command in Singularity 2.4 it said that the "create" command is depreciated, and image.create is recommended for this newer version of Singularity. Is there some point that I am missing?

Comment (vsoch) [reply README.md]: To be more clear, you can delete the line entirely. You don't need to create an image (with either method) before building.

Comment (hypotheses) [reply README.md]: Thanks for the explanation.

Comment (vsoch) [reply README.md]: :)

**Comment by vsoch** [comment]
Why did you close the pull request? The changes that you made were good! I was suggesting to amend the pr to remove that one line with the create command.

**Comment by hypotheses** [comment]
sorry. I'm new to this pull request/change thing. May be I've to close this first to make the change and request the pull request again?

**Comment by vsoch** [comment]
Oh I'd be happy to help you! You don't need to close it - you can just make more changes, commit to your branch, and push and the PR will be updated. Give it a try!

**Comment by hypotheses** [comment]
I've made some more changes. https://github.com/hypotheses/jupyter

**Comment by vsoch** [comment]
Did you mean to close it again?

**Comment by hypotheses** [comment]
It doesn't look like the changes have been updated here, but please let me know if I should create a new pull request.

**Comment by vsoch** [comment]
Are you sure? It looks good to me! Merging!

**Comment by hypotheses** [comment]
I guess I kept clicking on the old like above which shows me the previous commit that I submitted for the pull request. Is this the link that I can look for what is being sent https://github.com/singularityhub/jupyter/pull/1/commits? Or, will it always get the master from my repository?

**Comment by vsoch** [comment]
If you do a pull request from your master, while the pull request is still open, any updates that you push to your master will be represented here. As a sanity check you can look at the [files tab](https://github.com/singularityhub/jupyter/pull/1/files) to see the compiled changes. For the commit, you would want to click on the updated commit that you made, this link shown here:

image

**Pull request merged**